react/features/base/participants/reducer.js

```diff
@@ -215,10 +224,19 @@ ReducerRegistry.register('features/base/participants', (state = DEFAULT_STATE, a
             };
         }
 
-        state.remote.set(participant.id, participant);
+        state.remote.set(id, participant);
 
-        if (participant.isFakeParticipant) {
-            state.fakeParticipants.set(participant.id, participant);
+        // Insert the new participant.
+        const displayName = name ?? 'Fellow Jitser';
+        const sortedRemoteParticipants = Array.from(state.sortedRemoteParticipants);
+
+        sortedRemoteParticipants.push([ id, displayName ]);
+        sortedRemoteParticipants.sort((a, b) => a[1].localeCompare(b[1]));
+
+        state.sortedRemoteParticipants = new Map(sortedRemoteParticipants);
```

Comment:
Could you please add a comment specifying that the sort order will be preserved because the Map will keep the insertion order?

Reply:
Sure.